pressian.com.js

```diff
@@ -0,0 +1,32 @@
+function detectWeb(doc, url) { return FW.detectWeb(doc, url); }
+function doWeb(doc, url) { return FW.doWeb(doc, url); }
+
+
+FW.Scraper({
+itemType	: 'newspaperArticle',
+detect		: FW.Url().match(/\/news\/article\.html/),
+title		: FW.Xpath("//title/text()").text(),
+date		: FW.Xpath('//p[@class="arvdate"]/text()').text(),
+creators	: FW.Xpath('//p[@class="arvdate"]/a/span').text().replace(/ .*/,'').cleanAuthor('author'),
```

Comment:
Are you sure this is right? You're effectively only taking the first word in the span. Also, if the span starts with a space (not uncommon to have extra whitespace in HTML) you'd be clearing everything.